project/Dependencies.scala

```diff
@@ -9,7 +9,7 @@ import scala.util.Try
 object Dependencies {
   val scala212 = "2.12.18"
   val scala213 = "2.13.11"
-  val scala3 = "3.3.1-RC1"
+  val scala3 = "3.3.2-RC1-bin-SNAPSHOT"
```

Comment:
I am not familiar with the dotty project, but is there some immutable snapshot we will be able to use once the PR is merged?

Reply:
Yes, nightly builds should be available https://mvnrepository.com/artifact/org.scala-lang/scala3-compiler 👍